### PR TITLE
added command completion

### PR DESCRIPTION
**Pull request by necosta**
Hi @andreaTP , had some free time to explore the "completer ToDo", let me know what you think.

While on the interactive CLI, it looks to work nicely, on the command line, I'm just throwing a warning when no match

### CODE REVIEW

File: src/main/scala/org/akkajs/sbt/SbtClient.scala
Comment (andreaTP) [inline src/main/scala/org/akkajs/sbt/SbtClient.scala:125]: this warning is not worth given the fact that the list of available commands depends on the project definition itself.

File: src/main/scala/org/akkajs/sbt/SbtClient.scala
Comment (andreaTP) [inline src/main/scala/org/akkajs/sbt/SbtClient.scala:10]: why this rename?

File: src/main/scala/org/akkajs/sbt/SbtClient.scala
Comment (andreaTP) [inline src/main/scala/org/akkajs/sbt/SbtClient.scala:13]: this list has to come from sbt itself, we do not have any way to know this list of commands ahead of time.

This is some WIP for getting there: https://github.com/sbt/sbt/pull/4385

**Review by andreaTP — CHANGES_REQUESTED** [review]
I really appreciate this effort overall, but I truly think that this feature is ATM blocked by: https://github.com/sbt/sbt/pull/4385

File: src/main/scala/org/akkajs/sbt/SbtClient.scala
Comment (necosta) [reply src/main/scala/org/akkajs/sbt/SbtClient.scala:125]: Correct. Only changing sbt itself would allow all available commands (custom and base) to be available.

File: src/main/scala/org/akkajs/sbt/SbtClient.scala
Comment (necosta) [reply src/main/scala/org/akkajs/sbt/SbtClient.scala:10]: Class name and file name were normalised. Not a big deal...

File: src/main/scala/org/akkajs/sbt/SbtClient.scala
Comment (necosta) [reply src/main/scala/org/akkajs/sbt/SbtClient.scala:13]: I see... Now https://github.com/sbt/sbt/pull/4397 . Lets close this pull request and wait for developments in that front.

Comment (andreaTP) [reply src/main/scala/org/akkajs/sbt/SbtClient.scala:13]: Correct! as soon as we have that merged we can revamp this one :-)

**Pull request closed without merging**
